path_config.go

```diff
@@ -189,6 +197,22 @@ func (b *jwtAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Reque
 		ProviderConfig:       d.Get("provider_config").(map[string]interface{}),
 	}
 
+	// Check if the config already exists, to determine if this is a create or
+	// an update
```

Comment:
Might want to add a note on why we're doing this vs having an existence check.

Reply:
Adding something in 58a9841.